exchange/price_granularity.go

```diff
@@ -15,21 +16,27 @@ func GetCpmStringValue(cpm float64, config openrtb_ext.PriceGranularity) (string
 	bucketMax := 0.0
 	increment := 0.0
 	precision := config.Precision
-	// calculate max of highest bucket
+
+	// Limit the number of decimal significant figures. Very large values lead to Panics
+	if precision > 4 {
+		return cpmStr, fmt.Errorf("Limit the number of precision figures to 4. Parsed value: %d", precision)
+	}
+
 	for i := 0; i < len(config.Ranges); i++ {
-		if config.Ranges[i].Max > bucketMax {
+		// calculate max of highest bucket
```

Comment:
Consider removing this comment. I think the logic combined with the name `bucketMax` is self explanatory.